src/item/views/Item.tsx

```diff
@@ -263,85 +270,101 @@ const Item: FunctionComponent<ItemProps> = ({ history, match }) => {
 								<Container mode="vertical" size="small">
 									<Table horizontal untable>
 										<Grid tag="tbody">
-											<Column size="2-5" tag="tr">
-												<th scope="row">Publicatiedatum</th>
-												<td>{reorderDate(itemMetaData.issued || null, '/')}</td>
-											</Column>
-											<Column size="2-5" tag="tr">
-												<th scope="row">Toegevoegd op</th>
-												<td>{reorderDate(itemMetaData.published_at || null, '/')}</td>
-											</Column>
+											{!!itemMetaData.issued && (
+												<Column size="2-5" tag="tr">
+													<th scope="row">Publicatiedatum</th>
+													<td>{reorderDate(itemMetaData.issued, '/')}</td>
+												</Column>
+											)}
+											{!!itemMetaData.published_at && (
+												<Column size="2-5" tag="tr">
+													<th scope="row">Toegevoegd op</th>
+													<td>{reorderDate(itemMetaData.published_at, '/')}</td>
```

Comment:
nog correct formatteren
![image](https://user-images.githubusercontent.com/1710840/70262424-9c1d8380-1794-11ea-960c-1d8076e7ac67.png)

Reply:
Was al fixed in deze PR :)